Added iOS Text CocoaPods Podspec with -all_load linker flag

priankakariatyml opened the pull request:
1. Added target for building iOS Text Task Library static xcframework.
2. Added bash script to build CocoaPods archive of the xcframework.
3. Added podspec for Text Task Library.

schmidt-sebastian commented on mediapipe/tasks/ios/build_ios_framework.sh:
```diff
@@ -0,0 +1,153 @@
+# Set the following variables as appropriate.
+#   * BAZEL: path to bazel. defaults to the first one available in PATH
+#   * FRAMEWORK_NAME: name of the iOS framework to be built. Currently the
+#   * accepted values are TensorFlowLiteTaskVision, TensorFlowLiteTaskText.
```
Please update

schmidt-sebastian commented on mediapipe/tasks/ios/build_ios_framework.sh:
```diff
@@ -0,0 +1,153 @@
+BAZEL="${BAZEL:-$(which bazel)}"
+MPP_BUILD_VERSION=${MPP_BUILD_VERSION:-0.0.1-dev}
+MPP_ROOT_DIR=$(git rev-parse --show-toplevel)
+MEDIAPIPE_DISABLE_GPU=1
```
Can we make the default 0 and ensure that this is set to 0 for release builds?

schmidt-sebastian commented on mediapipe/tasks/ios/build_ios_framework.sh:
```diff
@@ -0,0 +1,153 @@
+MPP_BUILD_VERSION=${MPP_BUILD_VERSION:-0.0.1-dev}
+MPP_ROOT_DIR=$(git rev-parse --show-toplevel)
+MEDIAPIPE_DISABLE_GPU=1
+
```
It might be easier to parse if we add defaults for IS_RELEASE_BUILD, ARCHIVE_FRAMEWORK and DEST_DIR

schmidt-sebastian commented on mediapipe/tasks/ios/ios.bzl:
```diff
@@ -1,3 +1,40 @@
 
 MPP_TASK_MINIMUM_OS_VERSION = "11.0"
+
+# When the static framework is built with bazel, the all header files are moved
```
s/the all/all/

schmidt-sebastian commented on mediapipe/tasks/ios/ios.bzl:
```diff
@@ -1,3 +1,40 @@
+# to the "Headers" directory with no header path prefixes. This auxiliary rule
+# is used for stripping the path prefix to the C/iOS API header files included by
+# other C/iOS API header files.
+# In case of C header files includes start with a keyword of "#include'.
```
s/files/files,/

priankakariatyml commented:
@schmidt-sebastian Closing this PR since we are using the `force_load` option.  I have fixed all the review comments and used them to https://github.com/google/mediapipe/pull/4300. The PR also added the vision podspec and changes needed to make the vision and text pods coexist in the same iOS project. 
The PR does not include automated building of opencv XCFramework yet. I am still working on this.
